gitignore.js

```diff
@@ -72,7 +79,7 @@ const normalizeOpts = opts => {
 module.exports = o => {
 	const opts = normalizeOpts(o);
 
-	return globP('**/.gitignore', {ignore: opts.ignore, cwd: opts.cwd})
+	return fg('**/.gitignore', {ignore: opts.ignore.concat(DEFAULT_IGNORE), cwd: opts.cwd})
```

Comment:
Do we want to be able to overwrite the default ones? I guess putting `DEFAULT_IGNORE` at the end would take precedence over the users?